OoyalaSkinSampleApp/app/src/main/java/com/ooyala/sample/lists/AudioOnlyListActivity.java

```diff
@@ -0,0 +1,82 @@
+package com.ooyala.sample.lists;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.ListView;
+import com.ooyala.sample.R;
+import com.ooyala.sample.players.AudioOnlyPlayerActivity;
+import com.ooyala.sample.players.IntentBuilder;
+import com.ooyala.sample.utils.PlayerSelectionOption;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class AudioOnlyListActivity extends Activity implements AdapterView.OnItemClickListener {
```

Comment:
Isn't it better to extend from `AppCompatActivity` for better support?

Reply:
Copy paste from other samples

Reply:
Should we copy paste or make at least a new code better?

Reply:
Done